supervisely/project/project.py

```diff
@@ -4490,11 +4558,13 @@ async def _download_project_async(
                     only_image_tags=only_image_tags,
                     progress_cb=ds_progress,
                 )
-                tasks.append(task)
-            await asyncio.gather(*tasks)
+                await queue.put(task)
+
+        all_images = small_images + large_images
```

Comment:
is there a way to keep images order? (or it doesn't matter?)

Reply:
It doesn't matter for the following operations on the dataset. All images and annotations have already been saved before we start saving file metadata and deleting redundant items.